internal/pkg/manifest/validate.go

```diff
@@ -1282,3 +1319,10 @@ func validateWindows(opts validateWindowsOpts) error {
 	}
 	return nil
 }
+
+func validateARM(opts validateARMOpts) error {
+	if opts.Spot != nil || opts.SpotFrom != nil {
+		return errors.New(`'Fargate Spot' is not supported when deploying on ARM architecture`)
```

Comment:
nice validation!

Reply:
Oh interesting so windows doesn't support exec and efs whereas arm doesn't support spot 🤔